### PR TITLE
Front End Update

### PR DESCRIPTION
**Pull request by kdaeve**
Front end works, little bugs will be fixing later.

### CODE REVIEW

File: src/main/webapp/mainPage.html [hunk 1]
```diff
@@ -1,78 +1,60 @@
...
+        let matchToken = document.cookie.match(new RegExp('(^| )' + "token" + '=([^;]+)'));
+        if(matchToken!=null){
+            token = atob(matchToken[2]);
+        }
+        var url_string = window.location.href; 
+        var url = new URL(url_string);
+        var id = atob(url.searchParams.get("id"));
+        console.log(id);
+
+        var unRepliedResults = await fetch(("/display-posts-unreplied"), {
```
Comment (dmmmdg) [inline src/main/webapp/mainPage.html]: Are these changes from Constantine PR?

**Review by dmmmdg — APPROVED** [review]

File: src/main/webapp/mainPage.html (hunk 1 above)
Comment (kdaeve) [reply src/main/webapp/mainPage.html]: These changes should be coded by Constantine

File: src/main/webapp/mainPage.html [hunk 2]
```diff
@@ -24,14 +24,18 @@
         <input type="submit" value="Submit" className="submitBtn">
     </form>
     <hr>
-    <p id="unrepliedstyle">Unreplied Post:</p>
-    <button onclick="myFunction()">Expand/Collapse</button>
+    <p id="unrepliedstyle">Unreplied Post:
+        <button onclick="myFunction()" id="unrepliedbtn"> Expand/Collapse +</button>
```
Comment (dmmmdg) [inline src/main/webapp/mainPage.html:66]: can you rename the function to something more applicable.

Comment (dmmmdg) [reply src/main/webapp/mainPage.html:66]: something like collapsePosts() etc.

**Review by dmmmdg — APPROVED** [review]

**Pull request merged**